src/components/layout/header/AlarmList/index.tsx

```diff
@@ -0,0 +1,83 @@
+import Image from 'next/image';
+
+import { RefObject, useState } from 'react';
+
+import classNames from 'classnames/bind';
+
+import { MyNotifications } from '@/apis/myNotifications';
+import { SVGS } from '@/constants';
+import { getElapsedTimeToKST } from '@/utils';
+
+import { NotificationResponse } from '@/types';
+
+import styles from './AlarmList.module.scss';
+
+const cx = classNames.bind(styles);
+const { normal, hover } = SVGS.trashcan;
+
+type AlarmListProps = {
+  notifications: NotificationResponse[];
+  totalCount: number;
+  alarmListRef: RefObject<HTMLDivElement>;
+};
+
+export const AlarmList = ({ notifications, totalCount, alarmListRef }: AlarmListProps) => {
+  const [hoverStates, setHoverStates] = useState(Array(notifications.length).fill(false));
+
+  const handleToggleState = (id: number, value: boolean) => {
+    setHoverStates((prevStates) => {
+      const newStates = [...prevStates];
+      newStates[id] = value;
+      return newStates;
+    });
+  };
+
+  const handleDeleteNotification = (id: number) => {
+    MyNotifications.delete(id);
+  };
+
+  const handleDeleteAllNotifications = () => {
+    const notificationIds = notifications.map((notification) => notification.id);
+
+    notificationIds.forEach((id) => MyNotifications.delete(id));
+  };
```

Comment:
```suggestion
  const handleDeleteAllNotifications = () => {
    notifications.forEach(({id}) => MyNotifications.delete(id));
  };
```

요렇게 한줄로 줄일수 있을거같습니다!